views/style.css

```diff
@@ -76,18 +142,25 @@ a.navtab {
   flex-direction: column;
   text-align: center;
   color: white;
-  background: linear-gradient(#430577, #a3ffa0)
+  background: linear-gradient(to right, #8360c3, #2ebf91);
 }
 
 #kubercade_nexus h1 {
-  font-size: 32pt;
+  text-shadow: 4px 4px 2px rgb(22, 22, 22);
 }
 
 #kubercade_nexus h4 {
-  font-size: 24pt;
+  text-shadow: 2px 2px 2px rgb(22, 22, 22);
 }
 
 #kubercade_nexus a {
   font-size: 18pt;
   color: white;
-}
\ No newline at end of file
+}
+
+/*** chat ***/
+
+#chat_window {
+  background: linear-gradient(to top, rgb(64, 64, 64), rgb(128, 128, 128));
```

Comment:
I personally think that the screen looks a little busy with the chat as a gradient - solid black/white would probably fit in better.

Reply:
Fair enough, done with YouTube chat colors https://github.com/knative-portability/Kubercade/pull/13/commits/1103f8723d95e9330b6b5d9a1c37d95d68abf1d3